core/connection/manager.go

```diff
@@ -21,13 +21,16 @@ import (
 	"context"
 	"errors"
 	"sync"
+	"time"
 
 	log "github.com/cihub/seelog"
 	"github.com/mysteriumnetwork/node/communication"
 	"github.com/mysteriumnetwork/node/consumer"
 	"github.com/mysteriumnetwork/node/firewall"
 	"github.com/mysteriumnetwork/node/identity"
 	"github.com/mysteriumnetwork/node/market"
+	"github.com/mysteriumnetwork/node/money"
+	"github.com/mysteriumnetwork/node/services/openvpn/discovery/dto"
```

Comment:
Again I'm worried that we are using openvpn specific `dto.PaymentPerTime`.

Reply:
I'll create a separate ticket to move it to somewhere more appropriate.

Reply:
https://github.com/mysteriumnetwork/node/issues/761